tests/integration/routes/account/put-account-test.js

```diff
@@ -81,7 +81,7 @@ getServer(function (error, server) {
         .reply(201, [{
           id: 'org.couchdb.user:pat-doe',
           error: 'conflict',
-          reason: 'Document update conflict'
+          reason: 'An account with that username already exists'
```

Comment:
Ok, thanks. Should I change it back? Also could you look at my fix? Thank you for helping me with this.